examples/armv4t/gdb.rs

```diff
@@ -5,6 +5,20 @@ use gdbstub::{
 };
 
 use crate::emu::{Emu, Event};
+use core::convert::TryInto;
```

Comment:
tiiiny style nit: `core::` imports should be specified at the top of the file, and separately from external crate imports. Similarly, external crate imports should be separate from internal crate imports (notice blank line separating the two?).

So, move the core import up, and the `gdbstub` import up alongside the other `gdbstub` imports